conformance/Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright {yyyy} {name of copyright owner}
```

Comment:
@jdolitsky @imjasonh you seem to be the authors involved in this file. Could you add your copyright notice? If you are upstream maintainers, you should have the permission to do so. If you aren't please let me know so that I give you the permission.